meshnesh/react-consume-api
# Ft authorized routes edit events

# What does this PR do?

This PR allows the owner of an event to be able to edit his events

## How should this be manually tested?

Clone the repo.
Install the packages npm install then run npm run dev to run the app.
Create an account on the app by registering, confirm your email address.
Add an event, navigate to manage events.
Click edit on the current event to edit it.

# Screenshots

<img width="1280" alt="screen shot 2018-06-04 at 12 08 30" src="https://user-images.githubusercontent.com/15923109/40908913-cd6f0b52-67f0-11e8-952d-f556dc2b6392.png">
![Uploading Screen Shot 2018-06-04 at 12.09.00.png…]()
